.eslintrc.yml

```diff
@@ -0,0 +1,18 @@
+env:
+  browser: true
+  es6: true
+  mocha: true
+extends: 'eslint:recommended'
+globals:
+  Atomics: readonly
```

Comment:
Can you explain where `Atomics` and `SharedArrayBuffer` came from?

Reply:
@elizabethsjudd These were default globals set when the eslint config file was generated.

Reply:
@ChevonNwosu Could we try removing them and see if it still runs without errors?

Reply:
@elizabethsjudd Yeah, I'll give it a try